src/fckit/module/fckit_shared_ptr.F90

```diff
@@ -25,8 +25,7 @@ module fckit_shared_ptr_module
 #else
 use fckit_refcount_module, only : &
   & fckit_refcount, &
-  & fckit_refcount_interface, &
-  & fckit_external
```

Comment:
The removal of fckit_external is strange. Is this required?

Reply:
No, I have added it back.